src/components/contacts/SwipeableContactRow.js

```diff
@@ -83,8 +82,8 @@ const SwipeableContactRow = (
 
   const handleEditContact = useCallback(() => {
     swipeableRef.current?.close?.();
-    onSelectEdit({ address, color, ens, nickname });
-  }, [address, color, ens, nickname, onSelectEdit]);
+    onSelectEdit({ address, ens, nickname });
```

Comment:
the method passed down from props to this component is `handleEditContact` in `SendContactList`

```
const handleEditContact = useCallback(
    ({ address, ens, nickname, profile }) => { ... }
)
```
should we just remove `profile` from that method?

Reply:
i see this pr actually adds the `profile` object to that method https://github.com/rainbow-me/rainbow/pull/3464/files#diff-9adbfd266e01fe7faa82175d589a52b1d41601a82c042880839784ca10f055b5R98 but here we're not passing it